components/bitcoin-topics/Navigation.tsx

```diff
@@ -156,17 +311,25 @@ export function Navigation({
                                                                 href={
                                                                     childLink.href
                                                                 }
-                                                                onClick={
-                                                                    onLinkClick
-                                                                }
+                                                                onClick={(
+                                                                    e
+                                                                ) => {
+                                                                    e.stopPropagation()
+                                                                    handleTopicClick(
```

Comment:
`e.stopPropagation()` is used throughout the component to prevent clicks on the arrow from triggering the link click event. But Ithink it may not be needed in this speicific case. 
Removed here https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/102/commits/bb01fb7005ae14728d77f6afa66d485c56b6d766